src/Sql/Sql/help/Set-AzSqlServerAuditing.md

```diff
@@ -85,6 +85,7 @@ To use the cmdlet, use the *ResourceGroupName* and *ServerName* parameters to id
 The audit logs destination is determined by specifying one of the following switch parameters: BlobStorage, LogAnalytics or EventHub (if none is specified, the default is BlobStorage).
 Use the *State* parameter to enable/disable the policy.
 When audit logs destination is blob storage, specify the *StorageAccountName* parameter to determine the storage account for the audit logs and the *StorageKeyType* parameter to define the storage keys. You can also define retention for the audit logs by setting the value of the *RetentionInDays* parameter to define the period for the audit logs.
+To apply changes to the *AuditActionGroup* parameter, you must first disable auditing on all destinations and then reapply the configuration for each.
```

Comment:
You should be able to modify `AuditActionGroup` whether auditing is on or off. Notice that mentioned change affects all 3 targets.

Reply:
This was not my experience - I took a customer through resolving an issue trying to get the AuditActionGroup to change . . . the only way it would stick was to first disable all targets.  I will test again, but there would not have been need for the case had the configuration been applied effectively without this step.